dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/model/dataRequest/StoredDataRequest.kt

```diff
@@ -23,7 +23,7 @@ data class StoredDataRequest(
 
     val creationTimestamp: Long,
 
-    val dataType: DataTypeEnum?,
+    val dataType: DataTypeEnum,
 
     val reportingPeriod: String,
 
```

Comment:
Sollte `messageHistory` nicht nullable sein?
Es gibt doch durchaus data request Entitäten (v.a. alle, die aus einem bulk request stammen), die keine messageHistory haben.

Reply:
In dem Fall ist das dann eine leere Liste.

Reply:
Auch bei einer Bulk data request wird eine Message History ja erzeugbar sein, im Nachhinein, über die Request view page.

Reply:
Ah ok verstanden! Wusst ich nicht.  Quasi wie ein Facebook-Beitrag, der noch kein Kommentar hat, aber ja später noch eins bekommen könnte.
Ich schließe